modules/vaos/app/controllers/vaos/v2/slots_controller.rb

```diff
@@ -27,6 +29,7 @@ def facility_slots
           response = systems_service.get_available_slots(location_id:,
                                                          clinic_id: params[:clinic_id],
```

Comment:
get_available_slots was changed to accept options hash in systems_service.rb, but controller still calls it with named parameters